Sources/SwiftDocC/Checker/Checkers/TopicWithoutSubheading.swift

```diff
@@ -0,0 +1,72 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import Markdown
+
+/**
+ A `Topic` should have at least one subheading.
+ */
+public struct TopicWithoutSubheading: Checker {
+    public var problems = [Problem]()
+
+    private var sourceFile: URL?
+
+    /// Creates a new checker that detects Topics without subheadings.
+    ///
+    /// - Parameter sourceFile: The URL to the documentation file that the checker checks.
+    public init(sourceFile: URL?) {
+        self.sourceFile = sourceFile
+    }
+
+    public mutating func visitDocument(_ document: Document) -> () {
+        let headings = getHeadings(document: document)
+        for (index, element) in headings.enumerated() {
+            guard element.title == "Topics" else {
```

Comment:
Only level-2 headings are considered Topics sections so we should also check the level here.

Reply:
I wonder if it's worth offering a computed property on an extension of `Heading` to determine if it is a Topics section since I noticed this checker, `DuplicateTopicsSections,` and `NonOverviewHeadingChecker` all do the check.

I'm not sure, though, since it would be weird to only offer it for `Topics,` for example, and not another heading type, and I guess expanding it past `Topics` _could_ make it messy if we want to offer one for everything.

Reply:
Yes, a computed property could be good here. There are only a few headings with special meaning but Topics sections are the most common. The other headings types could add their own computed properties over time when they are checked for in more places.